src/ukify/ukify.py

```diff
@@ -1394,9 +1399,30 @@ def config_example(self) -> tuple[Optional[str], Optional[str], Optional[str]]:
 
 def apply_config(namespace, filename=None):
     if filename is None:
-        filename = namespace.config
+        if namespace.config:
+            # Config file specified, turn it into a pathlib.Path
+            filename = pathlib.Path(namespace.config)
+        else:
+            # If there is no config file provided, try to look for default configs and load the first found.
+            for config_dir in DEFAULT_CONFIG_DIRS:
+                config_path = pathlib.Path(config_dir).joinpath(DEFAULT_CONFIG_FILE)
+                if config_path.is_file():
+                    print(f'No config file specified but found config file: {config_path}')
+                    filename = config_path
+                    break
     if filename is None:
+        # No config file specified or found, nothing to do.
         return
+    # Also print the original filename if it is a symlink
+    # Too bad, pathlib.Path.resolve() has no option not to resolve symlinks
+    if filename.is_symlink():
+        print_filename = f'{filename.resolve()} (target of: {filename})'
+    else:
+        print_filename = f'{filename.resolve()}'
+    if filename.resolve().is_file():
+        print(f'Using config file: {print_filename}')
+    else:
+        raise FileNotFoundError(f'Config file not found or not a file: {print_filename}')
```

Comment:
Hmm, this whole part seems to be debug info that we isn't required for the normal user. Please drop it.